admob/src/common/admob_common.cc

```diff
@@ -50,6 +51,77 @@ DEFINE_FIREBASE_VERSION_STRING(FirebaseAdMob);
 static CleanupNotifier* g_cleanup_notifier = nullptr;
 const char kAdMobModuleName[] = "admob";
 
+const AdSize AdSize::kBanner(/*width=*/320, /*height=*/50);
```

Comment:
Can you reference where these come from?

Reply:
Added a comment!

Reply:
Which doesn't show up in the code snippet above.  I guess you have to go into review mode to see it.